lib/plugins/usermanager/lang/it/lang.php

```diff
@@ -66,8 +66,6 @@
 $lang['add_fail']              = 'Aggiunta utente fallita';
 $lang['notify_ok']             = 'Email di notifica inviata';
 $lang['notify_fail']           = 'L\'email di notifica non può essere inviata';
-$lang['import_userlistcsv']    = 'File lista utente (CSV):';
```

Comment:
If you experience undesired behaviour, and you can explain how to reproduce this, than it is useful for us if you create an issue at the issue tracker of the translation tool where you explain the details: https://github.com/dokufreaks/dokuwiki-translation/issues Thanks!